Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -505,7 +517,8 @@ ExceptionOr<void> MediaSource::clearLiveSeekableRange()
     // If the readyState attribute is not "open" then throw an InvalidStateError exception and abort these steps.
     if (!isOpen())
         return Exception { ExceptionCode::InvalidStateError };
-    m_private->clearLiveSeekableRange();
+    Ref msp = protectedPrivate().releaseNonNull();
+    msp->clearLiveSeekableRange();
```

Comment:
Why not simply protectedPrivate()->clearLiveSeekableRange();

Reply:
I believe this provides a clearer way to indicate that the RefPtr returned isn't null (as it's called after the check `(isClosed())`